torch/_torch_docs.py

```diff
@@ -1209,6 +1209,42 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.tensor_split,
+           r"""
+tensor_split(input, indices_or_sections, dim=0) -> List of Tensors
+
+Split a tensor into multiple sub-tensors, all of which are views of the
```

Comment:
"Splits ... all of which are views of :attr:\`input`. This function is based on NumPy's array_split."

For the ellipses, can the one-sentence description incorporate the required indices_or_sections argument? Maybe something as simple as "into multiple subtensors based on the indices or number of sections specified by :attr:\`indices_or_sections\'"?

Reply:
Done